### DISCUSSION TITLE
The organization has been flagged.

### DISCUSSION THREAD

**Discussion by adilAlphaisolutions** [community/community#30427] [New to GitHub] [answered]
My account has been flagged, can you please verify

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

The Community team is unable to assist with this account-specific matter.  The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact).

**Reply by vksarswat** [reply]
my account get flagged plz help me

**Reply by Warrenfox002** [reply]
My account has been flagged Please Help me.

**Reply by aikoons** [reply]
me too. my account has been flagged pls help me fix this

**Reply by Bipul-hossain** [reply]
my account is flagged pls help me

**Reply by ninja903** [reply]
my account is flagged as well so i am not able to use anything on github

**Comment by ghuy77** [comment]
我的帳戶已被標記，請您驗證一下

**Comment by ohld** [comment]
I just received the alert that several organizations were flagged. But I can't find which one was flagged. Can you please help? I also can't remove this annoying red banner with "your orgs were flagged" info

**Reply by queenofcorgis** [reply]
Hi there! The best place to go for answers to account/organization-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). The Community on Discussions team is unable to assist in matters like these. Best of luck!

**Comment by pavel-collab** [comment]
Hello, my organisation has been flagged without any cause. This is a student organisation with study content. We don't contact with government or any official organisation, we didn't break platform rules. We just wrote our lections by Latex and post it on our repositories. Could you pleas return the access to our organisation. Thank you!

**Reply by pachanixi** [reply]
Hi. Does anyone knows if there is a way of unflagging my account? I've submitted 2 tickets already without any response.

Thank you in advance.

**Comment by oxtrue-dev** [comment]
My account has been flagged, can you please verify

**Comment by ramzan2020** [comment]
My account has been flagged, please review & verify

**Comment by JoaoLucasPSP** [comment]
Minha conta foi sinalizada, poderia me ajudar

**Comment by wyphyoez** [comment]
Please help me,my organization has been flagged.why? @github-staff

**Reply by queenofcorgis** [reply]
@mrwaiyanphyoe Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by wyphyoez** [reply]
@queenofcorgis I have sent it.

**Reply by fishmandev** [reply]
Hi @queenofcorgis, I’ve already opened a support ticket about deleting my flagged organization, Steam-account, but it has been a long time and I still haven’t received a response.

Could you help get the ticket to the appropriate team or suggest how I can follow up? Thank you!
https://help.github.com/ticket/personal/0/4612988

**Reply by queenofcorgis** [reply]
Hi @fishmandev 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by veha-cgg** [comment]
How to solve this problem with github got flage?

**Comment by siliconvalleylocksmith01-cpu** [comment]
Hello. My account was flagged, and I cannot authorize Vercel to deploy my project. I am a new user and was just trying to host my website. Please review and unflag my account. Thank you.

**Comment by TrustDec** [comment]
Hello, Mr. or Ms.
I don’t understand what happened to my account. Would you tell me what to do now? Thank you very much for your support. I don’t know why my account has been flagged.
Why is my account 404? I didn‘t do anything illegal.
